notebooks/DrizzlePac/sky_matching/sky_matching.ipynb

```diff
@@ -4,14 +4,27 @@
    "cell_type": "markdown",
```

Comment:
<p>Instead of splitting the path manually to get the filename, use "os.path.basename"</p><p>Also:</p><p>Check if the path to "science" exists before removing it, as this will cause an error if run a second time.</p><p>Such:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">if os.path.exists(....):
   ...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>